File: app/features/document_intelligence/fixtures/simple.java
// Quarterly infrastructure report helper.

package com.example.infra;

public class RegionReport {
    private final String name;
    private final int nodes;
    private final double utilization;

    public RegionReport(String name, int nodes, double utilization) {
        this.name = name;
        this.nodes = nodes;
        this.utilization = utilization;
    }

    public boolean isHealthy() {
        return this.utilization < 0.85;
    }
}

File: app/features/document_intelligence/fixtures/simple.cs
// Quarterly infrastructure report helper.

namespace Example.Infra
{
    public class RegionReport
    {
        public string Name { get; }
        public int Nodes { get; }
        public double Utilization { get; }

        public RegionReport(string name, int nodes, double utilization)
        {
            Name = name;
            Nodes = nodes;
            Utilization = utilization;
        }

        public bool IsHealthy() => Utilization < 0.85;
    }
}
